code/modules/shuttle/dropship_hijack.dm

```diff
@@ -165,11 +163,8 @@
 /datum/dropship_hijack/almayer/proc/do_dropship_collision_sound()
 	playsound_z(SSmapping.levels_by_any_trait(list(ZTRAIT_MARINE_MAIN_SHIP)), 'sound/effects/dropship_crash.ogg', volume = 75)
 
-/datum/dropship_hijack/almayer/proc/disable_latejoin()
-	enter_allowed = FALSE
-
-/datum/dropship_hijack/almayer/proc/get_crashsite_turf(ship_section)
-	var/list/turfs = list()
+/datum/dropship_hijack/almayer/proc/get_crashsite_area(ship_section)
```

Comment:
You are currently reverting a change to a renamed proc here.

Reply:
Same as the initial var inside; I recommend dealing with conflicts with a 3 way merge tool (vsc supports this if you configure it) so you can view local, remote, and still have a panel to manually write changes that would combine the two blocks.